crates/miden-testing/src/tx_context/builder.rs

```diff
@@ -666,18 +666,20 @@ impl TransactionContextBuilder {
                     mock_chain.add_pending_note(OutputNote::Full(i));
                 }
 
-                mock_chain.prove_next_block();
-                mock_chain.prove_next_block();
+                mock_chain.prove_next_block().unwrap();
+                mock_chain.prove_next_block().unwrap();
 
                 let input_note_ids: Vec<NoteId> =
                     mock_chain.committed_notes().values().map(MockChainNote::id).collect();
 
-                mock_chain.get_transaction_inputs(
-                    self.account.clone(),
-                    self.account_seed,
-                    &input_note_ids,
-                    &[],
-                )
+                mock_chain
+                    .get_transaction_inputs(
+                        self.account.clone(),
```

Comment:
For a separate PR, should we also make the `TransactionContext(Builder)` types return errors and basically convert all tests that use it to also use `anyhow` or `miette` (depending on whether it executes MASM code via a `miden-vm` API, see also the PR description of https://github.com/0xMiden/miden-base/pull/1458).